.gitlab-ci.yml

```diff
@@ -8,7 +8,7 @@ variables:
     CLUSTERING_ARTIFACT_PATH: eva-accession-clustering/target/eva-accession-clustering-*.jar
     ACCESSIONING_ARTIFACT_PATH: eva-accession-pipeline/target/eva-accession-pipeline-*.jar
     RELEASE_ARTIFACT_PATH: eva-accession-release/target/eva-accession-release-*.jar
-    REMAPPING_GET_SOURCE_ARTIFACT_PATH: eva-remapping-get-source/target/eva-remapping-get-source-*.jar
+    REMAPPING_GET_SOURCE_ARTIFACT_PATH: eva-remapping-get-source/target/eva-remapping-get-source-*exec.jar
```

Comment:
don't understand why the `exec` suffix is needed

Reply:
we are generating two artifacts (2 jar files) for the **remapping_get_source_project** (as it is dependecy for eva-clustering project) and we need to pick the one with **exec** in the name.